noxfile.py

```diff
@@ -156,6 +156,7 @@ def default(session):
         "--cov-config=.coveragerc",
         "--cov-report=",
         "--cov-fail-under=0",
+        "-s",
```

Comment:
Assuming this will be removed before merge?

Reply:
I believe this will be removed. `noxfile.py` is not to be edited.